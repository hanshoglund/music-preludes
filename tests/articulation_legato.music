legato (scat [c..g]|/8)
