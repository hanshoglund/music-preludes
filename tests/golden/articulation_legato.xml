<?xml version='1.0' ?>
<score-partwise>
  <movement-title></movement-title>
  <identification>
    <creator type="composer"></creator>
  </identification>
  <part-list>
    <score-part id="P1">
      <part-name></part-name>
    </score-part>
  </part-list>
  <part id="P1">
    <measure number="1">
      <attributes>
        <key>
          <fifths>0</fifths>
          <mode>major</mode>
        </key>
      </attributes>
      <attributes>
        <divisions>768</divisions>
      </attributes>
      <direction>
        <direction-type>
          <metronome>
            <beat-unit>quarter</beat-unit>
            <per-minute>60</per-minute>
          </metronome>
        </direction-type>
      </direction>
      <attributes>
        <time>
          <beats>5</beats>
          <beat-type>8</beat-type>
        </time>
      </attributes>
      <attributes>
        <clef>
          <sign>G</sign>
          <line>2</line>
        </clef>
      </attributes>
      <note>
        <pitch>
          <step>C</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations />
          <slur number="1" type="start" />
        </notations>
      </note>
      <note>
        <pitch>
          <step>D</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
      </note>
      <note>
        <pitch>
          <step>E</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
      </note>
      <note>
        <pitch>
          <step>F</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
      </note>
      <note>
        <pitch>
          <step>G</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>384</duration>
        <voice>1</voice>
        <type>eighth</type>
        <notations>
          <ornaments />
          <technical />
          <articulations />
          <slur number="1" type="stop" />
        </notations>
      </note>
    </measure>
  </part>
</score-partwise>
